packages/manager/modules/netapp/src/dashboard/volumes/routing.js

```diff
@@ -0,0 +1,82 @@
+import Share from './Share.class';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('netapp.dashboard.volumes', {
+    url: '/volumes',
+    views: {
+      '@netapp.dashboard': {
```

Comment:
Isn't this the default behaviour ? 
```suggestion
```

Reply:
Yep you're right, old habit to define absolute path for views